### PR TITLE
4.14.0 initial release of updated specextract

### PR DESCRIPTION
**Pull request by nplee**
added support for the standard `ciao_contrib` parallelization parameters `parallel` and `nproc`.  ahelp file updated.

### CODE REVIEW

File: ciao_contrib/_tools/specextract.py [hunk 1]
```diff
@@ -109,6 +110,14 @@ def __init__(self,params):
         self.wmap_clip = params["wmap_clip"]
         self.wmap_threshold = params["wmap_threshold"]
 
+        if params["parallel"].lower() == "no":
```
Comment (DougBurke) [inline ciao_contrib/_tools/specextract.py:113]: I note that this logic is also in `bin/specextract` - I forget with the split up whether this is expected or not?

Comment (nplee) [reply ciao_contrib/_tools/specextract.py:113]: it is, there's some lightweight parallelization going on in here.  If we're alright with just forcing the parallelization and not trying to control it, I'd happily remove this logic.

File: bin/specextract [hunk 2]
```diff
@@ -1786,6 +1790,9 @@ def get_par(args):
     #pars["wmap_clip"] = params["wmap_clip"] = paramio.pgetstr(pfile,"wmap_clip")
     #pars["wmap_threshold"] = params["wmap_threshold"] = paramio.pgetstr(pfile,"wmap_threshold")
 
+    pars["parallel"] = params["parallel"] = paramio.pgetstr(pfile, "parallel")
+    pars["nproc"] = params["nproc"] = paramio.pgetstr(pfile, "nproc")
+
```
Comment (DougBurke) [inline bin/specextract:1795]: NOT FOR THIS PR, but I would really like to some day replace the strings in the parameter values with Python-appropriate data types - e.g. for `parallel` use bool and `nproc` use int.

File: ciao_contrib/_tools/specextract.py (hunk 1 above)
Comment (DougBurke) [reply ciao_contrib/_tools/specextract.py:113]: Let's leave that for a later cleanup.

**Pull request merged**